CMakeLists.txt

```
@@ -7,6 +7,9 @@ INCLUDE (cmake/openssl.cmake)
 OPTION (BUILD_BENCHMARK "Build benchmark" OFF)
 OPTION (BUILD_TESTS "Build tests" OFF)
 OPTION (WITH_OPENSSL "Use OpenSSL for TLS connections" OFF)
+OPTION (USE_SYSTEM_ABSEIL "Use system ABSEIL" OFF)
+OPTION (USE_SYSTEM_LZ4 "Use system LZ4" OFF)
+OPTION (USE_SYSTEM_CITYHASH "Use system cityhash" OFF)
```

```suggestion
OPTION (WITH_SYSTEM_ABSEIL "Use system ABSEIL" OFF)
OPTION (WITH_SYSTEM_LZ4 "Use system LZ4" OFF)
OPTION (WITH_SYSTEM_CITYHASH "Use system cityhash" OFF)
```
To make option names consistent